opentracing/tracer.py

```diff
@@ -82,11 +141,35 @@ def start_span(self,
             to avoid extra data copying.
         :param start_time: an explicit Span start time as a unix timestamp per
             time.time()
+        :param ignore_active_span: an explicit flag that ignores the current
+            active `Span` and creates a root `Span`.
 
         :return: Returns an already-started Span instance.
         """
         return self._noop_span
 
+    def start_span(self,
+                   operation_name=None,
+                   child_of=None,
+                   references=None,
+                   tags=None,
+                   start_time=None):
+        """DEPRECATED: this method is part of the previous API and must
+        not be used. Use `start_active_span()` or `start_manual_span()`
+        instead.
+        """
+        # TODO: print the WARNING only once otherwise it will flood
+        # users' logs. `warnings.simplefilter` helps on that.
+        warnings.warn('Deprecated!', DeprecationWarning, stacklevel=2)
```

Comment:
The idea is to use a `warnings.simplefilter` so that this deprecation is printed only once. It requires a custom `DeprecationWarning` that we may put in another module like `deprecations.py`.